### ISSUE TITLE
Fix spawning for Chocobos.

### ISSUE THREAD

**Issue by Tokeli** [Slayer5934/ChocoCraft3SLY#19]
I loaded the mod up in a fresh new MC install and spawned in an Extreme Hills biome, and there are chocobos everywhere you turn. I don't know whether it's intended or not, but they were the only things spawning there, and every Extreme Hills is filled with them.

![image](https://user-images.githubusercontent.com/6316563/43985872-fd06b758-9cd8-11e8-982a-6f8f12e4ae2b.png)

**Comment by SlayerTheChikken** [comment]
Interesting, Ill look into this, I recently added them to the hills biome list.

**Comment by Tokeli** [comment]
I've been playing with this mod for a couple days now in my pack (it was tested originally on a completely fresh MC) and they've been spawning normally (fairly rare, but they're around, which keeps it nice and rewarding) in every other biome. They just spew everywhere in Extreme Hills.

**Comment by SlayerTheChikken** [comment]
Yeah it may just be I need to adjust the "weight"/chance of spawning, thank you for the report

**Comment by SlayerTheChikken** [comment]
I couldnt even find a single chocobo generating in the world.
Needless to say ill be looking further into this issue. There may be something wrong with world gen.

**Comment by SlayerTheChikken** [comment]
I've looked and I've looked, I changed some code, it may have changed a little.

I can't exactly pinpoint the issue, I think its just minecraft to be honest. I went ahead and made a custom world will extreme hills only and I have found that sometimes chickens weren't spawning, sometimes the world was littered with chickens, sometimes littered with chocobos, sometimes it was mixed perfectly; Honestly I don't think this is a bug, just how strangely minecraft handles spawning.

But if your modpack is having constant spam of chocobos, there's a setting in the config called chocoboSpawnWeight, if you adjust this down less should spawn.
